internal/handlers/common/aggregations/stages/projection/projection.go

```diff
@@ -62,21 +63,17 @@ func ValidateProjection(projection *types.Document) (*types.Document, bool, erro
 			return nil, false, lazyerrors.Error(err)
 		}
 
-		if strings.HasPrefix(key, "$") {
```

Comment:
The operators (standard) both work for aggregation project and query projection. So what's our plan for supporting both? Either way I think we shouldn't divert the code if we know that they both work. :hugs: 
```
test> db.Composites.find().projection({type: {$type: "$v"}}).limit(3)
[
  { _id: 'array-documents', type: 'array' },
  { _id: 'document', type: 'object' },
  { _id: 'document-composite-numerical-field-name', type: 'object' }
]
test> db.Composites.aggregate([{ $project: { type: { $type: "$v" } } }, {$limit: 3}])
[
  { _id: 'array-documents', type: 'array' },
  { _id: 'document', type: 'object' },
  { _id: 'document-composite-numerical-field-name', type: 'object' }
]
```

Reply:
Let's ignore this comment.